Formula/eris.rb

```diff
@@ -20,8 +20,11 @@ def install
 
     (buildpath/"src/github.com/eris-ltd").mkpath
     ln_sf buildpath, buildpath/"src/github.com/eris-ltd/eris-cli"
-
     system "go", "build", "-o", "#{bin}/eris", "github.com/eris-ltd/eris-cli/cmd/eris"
+
+    man1.mkpath
+    File.write("#{man1}/eris.1", `#{bin}/eris man --dump`)
+    man1.install
```

Comment:
What does nothing `man1.install` or `man1.mkpath`? Without `man1.install` the man page doesn't end in the `/usr/local/share/man/man1`.